aws/README.md

```diff
@@ -1 +1,4 @@
-#AWS tools
+# AWS tools
+
+## Delete S3 velero bucket
+This script deletes a bucket associated with a cluster after the cluster is uninstalled.
```

Comment:
This script will delete the bucket regardless the cluster state (installed/uninstall) right? 
so `after the cluster is uninstalled.` is not right.